### ISSUE TITLE
`cuddled-lists` does not work with a single list item

### ISSUE THREAD

**Issue by blitzmann** [trentm/python-markdown2#286]
Hi there!

```
This is a list with a single item:
-  Single item
```

markdown2 does not interpret this as a list it doesn't seem, even with the `cuddled-lists` extension. If you add another element, it seems to work fine.

**Comment by carloseaf** [comment]
Hi,

Is there any updates about this issue?

**Comment by nicholasserra** [comment]
Hello,

Nope, no update.  PRs welcome for this though, thanks!

**Comment by blitzmann** [comment]
Looking at the source, it would seem this is expected behavior. Can you
provide a bit of commentary behind how it really should function?

**Comment by nicholasserra** [comment]
I'm not totally sure if there's any specifics in the markdown spec that would tell whether a single item list should work. I would personally expect the list syntax to work even for a single item list. I'd check what other packages and sites do.

**Comment by blitzmann** [comment]
it works with single lists if it's in this syntax:

```
This is a list with a single item:

-  Single item
```

(a newline at the top of the list)

**Comment by hmagdy** [comment]
https://github.com/trentm/python-markdown2/pull/288

**Comment by nicholasserra** [comment]
Closed via #288
